fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,246 @@
+# Multi-cluster configuration with Fleet Packages
+
+[Fleet Packages] provides a way to deploy configuration into multiple clusters in a [fleet].
+
+## Prerequisistes
+
+1. [gcloud CLI], as this guide uses it to set up the necessary infrastructure and for using the Fleet Packages service.
+2. A GitHub account, as we will fetch the configuration from git. Authentication to git will be set up using a GitHub App, so you must have admin-level permissions on your GitHub repository.
+3. A Google Cloud project with billing and with GKE Enterprise enabled.
+
+
+## Configure the environment
+
+Create an environment variable in your shell that identifies the project you want to use:
+
+```
+PROJECT_ID=<MY_PROJECT>
+```
+
+Also set the current project in gcloud to the same project:
+
+```
+gcloud config set project $PROJECT_ID
+```
+
+If you want to use your current project, you can set the environment variable with:
+
+```
+PROJECT_ID=$(gcloud config get-value project)
+```
+
+## Install Config Sync across the fleet
```

Comment:
Does this step need to happen after the fleet exists?

Reply:
I tested this guide on a brand new project and it works without creating the fleet first.